build/common/installer/scripts/tomlparser.rb

```diff
@@ -25,6 +25,9 @@
 @adxDatabaseName = "containerinsights" # default for all configurations
 @logEnableMultiline = "false"
 @stacktraceLanguages = "go,java,python,dotnet"
+@logEnableKubernetesMetadata = false
+@logKubernetesMetadataiIncludeFields = "podLabels,podAnnotations,podUid,image"
```

Comment:
we should also add support for "userPodLabels" and "systemPodLabels", "userPodAnnotations" and "systemPodAnnotations". Customers might prefer to exclude "systemPodLabels" for optimizing their cost.

Reply:
will go after private preview